src/ert/run_models/manual_update.py

```diff
@@ -19,6 +19,7 @@
 
 class ManualUpdateConfig(UpdateRunModelConfig):
     ensemble_id: str
+    ert_templates: list[tuple[str, str]]
```

Comment:
Is this needed? (asking cause it was not needed in 39a32cb0829186b61c5cecc4fc32b0bd74fde524 ).

It is also defined in InitialEnsembleRunModelConfig